app/forms/portal/client_login_form.rb

```diff
@@ -0,0 +1,5 @@
+module Portal
+  class ClientLoginForm < Form
+    attr_accessor :last_four_ssn, :confirmation_number, :token
```

Comment:
curious if this will be blacklisted successfully by our  sensitive fields blacklist